src/main/java/com/kaleido/cabinet/client/CabinetClientConfiguration.java

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
We don't need the copyright header since there is a License.md file stating the fair use of this project at the root of the project